doc/compliancetest.rst

```diff
@@ -405,45 +413,106 @@ Load PLC program
 Verify that the sample application PLC program is working properly with your
 IO-device. Button1 should be able to control the state of data LED (LED1).
 
+Interoperability
+^^^^^^^^^^^^^^^^
+Run with PLC for 10 minutes without errors.
+If the device under test has more than one port, there should be 5 IO-devices
+connected to the non-PLC port.
+
+The timing should be the fastest allowed according to the GSDML
+file, and use 3 "accepted update cycles without IO data".
+Record startup and data exchange using Wireshark.
+
+In the Wireshark file, make sure IOPS and IOCS in the cyclic data from the
+IO-device have the value GOOD after it has sent the "application ready"
+message.
+Also verify that there has been no alarms (sort the frames by protocol).
+
+* "Record data"?
+* ExpectedIdentification is equal to the RealIdentification?
+* How to create additional net load? (using DCP Identify all)
+* Implicit read?
 
 Data Hold Timer
 ^^^^^^^^^^^^^^^
-Unplug network cable from the PLC while recording the cyclic data (use Wireshark).
-Count the number of cyclic data frames before the alarm from the IO-device is sent.
-It is allowed that 3-6 data frames are sent before sending the alarm frame.
+Run with PLC. The timing should be the fastest allowed according to the GSDML
+file, and use 3 "accepted update cycles without IO data".
+Record startup and data exchange using Wireshark.
 
-Perform the cable unplugging measurements with reduction ratios 1, 2, 4, 8 and 16.
-The data hold time should be 3x the frame interval.
-With a cycle time of 1 ms this corresponds to a frame send interval of
-1 ms to 16 ms, and a data hold time of 3 ms to 48 ms.
+Unplug network cable from the PLC.
 
-In Siemens TIA portal, set the update time to for example 2 ms (values
-1, 2, 4, 8, and 16 ms should be used).
-The "Accepted update cycles without IO data" should be set to 3.
+In the Wireshark file:
 
-At startup the first valid data frame should be sent within the data hold time.
-Check IOPS.
+* Count the number of cyclic data frames sent by the IO-device before the
+  alarm frame appears. It is allowed that 3-6 data frames are sent before
+  the alarm frame.
+* At startup the first valid data frame should be sent within the data
+  hold time.
+* The IOCS in the cyclic data from the IO-device have the value GOOD after
```

Comment:
_should_ have the value GOOD?

Missing closing ".

Reply:
```
The IOCS in the cyclic data from the IO-device should have the value GOOD
after the "application ready" message has been sent.
```
Done